config.c

```diff
@@ -877,6 +878,7 @@ int readAllConfigPaths(const char **paths)
         .oldDir = NULL,
         .criterium = ROT_SIZE,
         .threshold = 1024 * 1024,
+        .minutes = 0,
```

Comment:
Copying `minutes` in `copyLogInfo()` is missing.

Reply:
done